eng/references.targets

```diff
@@ -55,8 +74,10 @@
     </TargetPathWithTargetPlatformMoniker>
   </ItemDefinitionGroup>
 
-  <Target Name="ValidateReferenceAssemblyProjectReferences" Condition="'$(IsReferenceAssembly)' == 'true'" AfterTargets="ResolveReferences">
-    <Error Condition="'%(ReferencePath.ReferenceSourceTarget)' == 'ProjectReference' AND '%(ReferencePath.IsReferenceAssembly)' != 'true' AND '%(ReferencePath.ReferenceAssembly)' == ''"
+  <Target Name="ValidateReferenceAssemblyProjectReferences"
+          AfterTargets="ResolveReferences"
+          Condition="'$(IsReferenceAssembly)' == 'true'">
+    <Error Condition="'%(ReferencePath.ReferenceSourceTarget)' == 'ProjectReference' and '%(ReferencePath.IsReferenceAssembly)' != 'true' and '%(ReferencePath.ReferenceAssembly)' == ''"
```

Comment:
This is just updating the statement to use lowercase "AND" and put the condition and aftertargets attribute into subsequent lines instead of everything in one line.